flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionBuilder.java

```diff
@@ -112,6 +114,11 @@ public ResultPartitionBuilder setBufferPoolFactory(
 		return this;
 	}
 
+	public ResultPartitionBuilder setIsExternallyManaged(boolean isExternallyManaged) {
```

Comment:
could be `isExternallyManaged()` which sets `this.isExternallyManaged = true`. This makes the API a bit conciser.